internal/condition/condition.go

```diff
@@ -14,10 +14,12 @@ type Condition interface {
 	Eval(object interface{}) (bool, error)
 }
 
+// Conditions defines list of Condition instances by their names
```

Comment:
It is not a list, but a map. You cannot define "a list by their names". You can key a set  of conditions against their name.